docker/travis_test/run.sh

```diff
@@ -2,11 +2,20 @@
 
 set -e
 
+INSTALL_FROM_CPAN="${INSTALL_FROM_CPAN:-0}"
+
 # Prepare dir and chdir into it before executing the wanted action
 sudo cp -rd /opt/repo /opt/run
 sudo chown -R $NORMAL_USER:users /opt/run
 
 pushd /opt/run
+[ "$INSTALL_FROM_CPAN" -eq 1 ] && \
```

Comment:
So for now this will be deactivated right?

Reply:
Yup, it's still bit tricky to use if you run the tests from Makefile, but if you use docker run manually, you need to supply -e INSTALL_FROM_CPAN=1 to get there